src/OfficeService.Data.Provider.MsSql.Ef/OfficeServiceDbContext.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Reflection;
+using System.Threading.Tasks;
+using LT.DigitalOffice.OfficeService.Models.Db;
+using Microsoft.EntityFrameworkCore;
+
+namespace LT.DigitalOffice.OfficeService.Data.Provider.MsSql.Ef
+{
+  /// <summary>
+  /// A class that defines the tables and its properties in the database of OfficeService.
```

Comment:
Do we need this comment?

Reply:
it is documentation, so I decided don't remove this